stablehlo/dialect/StablehloOps.cpp

```diff
@@ -1428,53 +1428,8 @@ LogicalResult RealDynamicSliceOp::reifyReturnTypeShapes(
 // Checks that the result type is of the form `zero_or_more_type(s),
 // stablehlo::token`
 LogicalResult InfeedOp::verify() {
```

Comment:
Let's also move the comment to TypeInference.cpp?

Reply:
Fixed.